source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -382,6 +382,24 @@ def __init__(self):
             hidden=True,
             help="how long it takes to park the car, " "used especially in distributed scenario",
         )
+        parser_get.add_argument(
+            "_here_realtime_traffic",
+            type=six.text_type,
+            hidden=True,
+            help="Here, Active or not the realtime traffic information (enabled/disabled)",
+        )
+        parser_get.add_argument(
+            "_here_matrix_type",
+            type=six.text_type,
+            hidden=True,
+            help="Here, street network matrix type (simple_matrix/multi_direct_path)",
+        )
+        parser_get.add_argument(
+            "_here_max_matrix_points",
```

Comment:
this parameter is actually defined by https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/interfaces/v1/Journeys.py#L365, I think you don't have to set here...

Reply:
I created this parameter, with the prefix _here, for debug purpose.
It is specific to Here because it is limited between 0 and 100